### ISSUE TITLE
Swiping left or right frequently seemingly does not do anything (until screen update)

### ISSUE THREAD

**Issue by zsszatmari** [nylas/N1#1862]
The problem is when I slide an item to left or right (single panel) mode, it frequently just slides back, instead of archiving or snoozing. (In the snooze case it lets me select proper time period but then slide back anyway).
Only after navigating away to other folder and back (or see a mail and go back) the email in question is correctly removed from list.
Thanks for looking into this!

##### Are there any related issues?

No, AFAIK.

##### What operating system are you using?

OS X El 10.11.4

##### What version of N1 are you using?

0.4.19-d41e72c

##### Do you have any third-party plugins installed?

No.

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?

Don't really know because I am using only Gmail.

##### Is the issue reproducible with a particular attachment, message, signature, etc?

No. It's more like N1 sometimes gets in a spooky mood.

##### Does this feature exist in another mail client or tool you use?

N/A

**Comment by jackiehluo** [comment]
Hey! Thanks for reporting. Looks like it's the same issue as #1385, so merging it into there!

**Comment by zsszatmari** [comment]
Okay, thanks!

**Comment by jstejada** [comment]
referencing #1720

**Comment by jstejada** [comment]
Hey @zsszatmari —this should be fixed in 0.4.32, which shipped Monday. Feel free to file a new ticket if you are still having issues with the thread list not reflecting your actions properly.
